src/core/src/package_managers/YumPackageManager.py

```diff
@@ -282,7 +287,121 @@ def is_package_version_installed(self, package_name, package_version):
+        # kernel-modules   x86_64  4.18.0-372.64.1.el8_6 rhel-8-for-x86_64-baseos-eus-rhui-rpms   32 M
+        #
+        #Transaction Summary
+        #=============================================================================================
+        #Install  2 Packages
+        #
+        #Total download size: 72 M
+        #Installed size: 93 M
+        #Operation aborted.
+        
+        # In sample1, the starting delimeter is "Upgrading:" and in sample2, the starting delimeter is "Installing dependencies:"
+        # If there are combinations of updates and new installations, one of the above delimeters will come first and the below logic will work.
+        
+        # There is one edge scenario which is also handled in the below parsing logic i.e., the package detail spills over to two lines:
+        # Sample output for the cmd 'sudo yum update --assumeno polkit.x86_64' is :
+        #
+        #Last metadata expiration check: 0:08:47 ago on Tue 25 Jul 2023 02:14:28 PM UTC.
+        #Package polkit-0.115-13.el8_5.2.x86_64 is already installed.
+        #Dependencies resolved.
+        #================================================================================
+        # Package   Arch   Version          Repository                              Size
+        #================================================================================
+        #Upgrading:
+        # polkit    x86_64 0.115-14.el8_6.1 rhel-8-for-x86_64-baseos-eus-rhui-rpms 154 k
+        # polkit-libs
+        #           x86_64 0.115-14.el8_6.1 rhel-8-for-x86_64-baseos-eus-rhui-rpms  77 k
+        #
+        #Transaction Summary
+        #================================================================================
+        #Upgrade  2 Packages
+        #
+        #Total download size: 231 k
+        #Operation aborted.
+
+        package_extensions = ["x86_64", "noarch", "i686"]
+        dependencies = []
+        lines = output.strip().split('\n')
+        upgrading_or_installing_flag = False
+
+        for line_index in range(0, len(lines)):
+            line = lines[line_index]
+            if not upgrading_or_installing_flag:
+                if not line.startswith("Upgrading:") and not line.startswith("Installing dependencies:"):
+                    self.composite_logger.log_debug(" - Inapplicable line: " + str(line))
+                    continue
+                else:
+                    upgrading_or_installing_flag = True
+                    self.composite_logger.log_debug(" - Packages list started: " + str(line))
+                    continue
+
+            if line.startswith("Transaction Summary"):
+                break
+
+            updates_line = re.split(r'\s+', line.strip())
+            dependent_package_name = ""
+
+            if len(updates_line) == 6 and updates_line[1] in package_extensions:
+                dependent_package_name = updates_line[0] + "." + updates_line[1]
```

Comment:
Shouldn't this be: dependent_package_name = updates_line[1] + "." + updates_line[2]? 

Since updates_line[1] is the package name and updates_line[2] Arch